lib/tasks/reminder.rake

```diff
@@ -0,0 +1,24 @@
+namespace :reminder do
+  desc "Send actionable approver reminder email for pending requests"
+  task actionable_approvers: :environment do
+    proposals = Proposal.pending.where(client_data_type: "Ncr::WorkOrder")
+    proposals.each do |proposal|
+      send_reminder_email(proposal)
+    end
+  end
+
+  private
+
+  def send_reminder_email(proposal)
+    user = proposal.client_data.current_approver
+    unless user
```

Comment:
It would be also possible to do `return unless user = proposal.client_data.current_approver`, no?